website/source/docs/guides/connect-gateways.md

```diff
@@ -0,0 +1,421 @@
+---
+layout: "docs"
+page_title: "Connecting Services Across Datacenters"
+sidebar_current: "docs-guides-connect-gateways"
+description: |-
+  Connect services and secure inter-service communication across datacenters
+  using Consul Connect and mesh gateways.
+---
+## Introduction
+
+Consul Connect is Consul’s service mesh offering, which allows users to observe
+and secure service-to-service communication. Because Connect implements mutual
+TLS between services, it also enabled us to build mesh gateways, which provide
+users with a way to help services in different datacenters communicate with each
+other. Mesh gateways take advantage of Server Name Indication (SNI), which is an
+extension to TLS that allows them to see the destination of inter-datacenter
+traffic without de-encrypting the message payload.
+
+Using gateways for inter-datacenter communication can prevent each Connect proxy
```

Comment:
```suggestion
Using mesh gateways for inter-datacenter communication can prevent each Connect proxy
```

Or you could move up the bit below that clarifies that within this doc "gateway" means mesh gateway